src/test/skript/tests/syntaxes/expressions/ExprBlocks.sk

```diff
@@ -0,0 +1,17 @@
+test "blocks void":
+	set {_loc} to location(0.5, 320.5, 0.5)
+	set {_blocks::*} to blocks between {_loc} and ({_loc} ~ vector(10,0,0))
+	assert size of {_blocks::*} is 11 with "Blocks between loc and (loc~vector(10,0,0)) is not 11"
+	assert blocks at {_blocks::*} is void air with "Blocks can be set in the void?"
+	set blocks at {_blocks::*} to stone
```

Comment:
need to clean up behind yourself

Reply:
I'm not sure wym? The intention behind this part is to ensure that the error is fixed from the linked issue, as well as to check if blocks in the void are not void air.

Reply:
ah right this one doesn't actually change anything